Fix:

implicitly declaring library function 'isalnum' with type 'int (int)'

implicitly declaring library function 'memcpy' with type 'void *(void *, const void *, unsigned long)'

implicitly declaring library function 'memset' with type 'void *(void *, int, unsigned long)'

implicit declaration of function 'mkdir' is invalid in C99

implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)'
--- clit/display.c.orig	2004-07-02 07:19:58.000000000 -0500
+++ clit/display.c	2020-10-06 18:09:32.000000000 -0500
@@ -22,6 +22,7 @@
 */
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include "litlib.h"
 
 extern void hexdump( unsigned char * ptr, int size );
--- clit/drm5.c.orig	2004-07-04 16:19:32.000000000 -0500
+++ clit/drm5.c	2020-10-06 18:16:12.000000000 -0500
@@ -26,6 +26,7 @@
  * Note that the July 4th update added only a trivial amount of complexity
  * to this process. 
  */
+#include <ctype.h>
 #include <sys/types.h>
 #include <stdio.h>
 #include <stdlib.h>
--- clit/explode.c.orig	2004-07-04 06:33:52.000000000 -0500
+++ clit/explode.c	2020-10-06 18:11:47.000000000 -0500
@@ -24,6 +24,7 @@
 #include <stdio.h>
 #include <stdlib.h> 
 #include <string.h>
+#include <sys/stat.h>
 #include "clit.h"
 #include "litlib.h"
 #include "manifest.h"
--- clit/hexdump.c.orig	2004-05-14 23:09:14.000000000 -0500
+++ clit/hexdump.c	2010-11-01 17:46:28.000000000 -0500
@@ -16,6 +16,7 @@
 #include <ctype.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 
 typedef unsigned char byte;
 
--- clit/manifest.c.orig	2004-07-04 06:02:56.000000000 -0500
+++ clit/manifest.c	2020-10-06 18:02:23.000000000 -0500
@@ -23,6 +23,7 @@

 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include "litlib.h"
 #include "manifest.h"
 #include "utils.h"
--- clit/transmute.c.orig	2003-07-05 19:40:14.000000000 -0500
+++ clit/transmute.c	2020-10-06 18:01:54.000000000 -0500
@@ -22,6 +22,7 @@
 */
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include "litlib.h"
 
 extern char * writingFilename;
--- lib/newlzx/lzxglue.c.orig	2004-07-04 09:38:58.000000000 -0500
+++ lib/newlzx/lzxglue.c	2020-10-06 18:13:48.000000000 -0500
@@ -28,6 +28,7 @@
  */
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include "litlib.h"
 #include "mspack.h"
 #include "lzx.h"
